_posts/2018-09-29-analyzing-the-behaviour-of-python-function-splice.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "Analyzing the behaviour of Python function slice"
+date: "2018-09-29 10:17:46 +0530"
+tag:
+  - python
+  - slice
+  - python trick questions
+---
+
+
+![Title Image](/assets/images/python_slice_function/title_image.jpg)
+
+Last Friday, I was sitting at one of the good coffee shop of Bangalore with my
+friend. Coffee and discussion is the best combination to release stress. It was
+looking like a perfect Friday eve, until my friend stroke with an idea of asking
```

Comment:
Friday evening, do you mean? (Friday eve would be the day before Friday, and mostly an archaic term at this point.)